packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -79,6 +79,52 @@ const CommonFinders find = CommonFinders._();
 /// See also [FlutterDriver.waitFor].
 typedef EvaluatorFunction = dynamic Function();
 
+/// Creates a FlutterDriver instance.
+typedef DriverCreatorFunc = Future<FlutterDriver> Function(
```

Comment:
Probably keep the named parameter for readability.

Also I'm concerned about this being a breaking change when we ever need to add a new parameter. Do you think it's worth making it to take an object instead? Like

```dart
class FlutterDriverCreatorParameter {
    String dartVmServiceUrl;
    bool printCommunication;
    bool logCommunicationToFile;
    int isolateNumber;
    Pattern fuchsiaModuleTarget;
    bool browser;
    Duration timeout;
}

typedef DriverCreatorFunc = Future<FlutterDriver> Function(FlutterDriverCreatorParameter);
```

cc @jonahwilliams , what do you think?